src/api/code-challenge/content-types/code-challenge/test-evaluator.ts

```diff
@@ -0,0 +1,102 @@
+import {
+  evaluateWithContext,
+  getCode,
+  getEvaluationContext,
+  overrideConsoleLog,
+  restoreConsoleLog,
+} from "../code-challenge/utils";
+
+type runTestEvaluatorProps = {
+  metaCaseCode: string;
+  internalTestCode: string;
+  metaLabel: string;
+  challengeLabel: string;
+  metaTestId: number;
+  evalResultShouldBe: boolean;
+  internalTest: { id: number };
+  removeComments?: boolean;
+};
+
+export const runTestEvaluator = async ({
+  metaCaseCode,
+  internalTestCode,
+  metaTestId,
+  evalResultShouldBe,
+  internalTest,
+  removeComments,
+}: runTestEvaluatorProps) => {
+  const setShortError = () => {
```

Comment:
so now you wouldn't preface these names with `set`, you would preface them with `get`

Reply:
ok I changed the names